pyomo/contrib/parmest/parmest.py

```diff
@@ -229,11 +229,119 @@ def _experiment_instance_creation_callback(
 
 def SSE(model):
     """
-    Sum of squared error between `experiment_output` model and data values
+    Sum of squared error between the model prediction of measured variables and data values,
+    assuming Gaussian i.i.d errors.
     """
-    expr = sum((y - y_hat) ** 2 for y, y_hat in model.experiment_outputs.items())
+    expr = sum((y - y_hat) ** 2 for y_hat, y in model.experiment_outputs.items())
     return expr
 
+def SSE_weighted(model):
+    """
+    Weighted sum of squared error between the model prediction of measured variables and data values,
+    assuming Gaussian i.i.d errors.
+    """
+    expr = (1 / 2) * sum(
+        ((y - y_hat) / model.measurement_error[y_hat]) ** 2
+        for y_hat, y in model.experiment_outputs.items()
+    )
+    return expr
+
+# Calculate the sensitivity of measured variables to parameters using central finite difference
+def _compute_jacobian(model, relative_perturbation, solver_option="ipopt"):
+    """
+    Computes the Jacobian matrix using central finite difference scheme
```

Comment:
Jacobian matrix of measurement predictions with respect to changes in parameter values.